### PR TITLE
Improved %help magic.

### PR DESCRIPTION
**Pull request by mcaceresb**
- If the environment cannot display HTML, %help directs the user to the
  online documentation; otherwise it displays the following message:

    stata_kernel 1.4.2 for stata 15.1. Type

        %help kernel

    for help on using the kernel and

        %help magics

    for info on magics. To fetch HTML help for a Stata command type

        %help command_or_topic

- %help kernel and %help magics show HTML help based on the online docs.
- Run `cd stata_kernel/docs && bash make.sh` to refresh

- [ ] See #106
- [ ] tests added / passed
- [X] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by kylebarron** [comment]
@mcaceresb 
There's a lot of leading whitespace. I think it would look cleaner without the extra whitespace. Is this a CSS issue?

image
image

File: stata_kernel/docs/css/pandoc.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+img {
+  max-width: 75%;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  /* color: #111; */
+  line-height: 125%;
+  margin-top: 2em;
```
Comment (kylebarron) [inline stata_kernel/docs/css/pandoc.css]: Changing this to `0em` looks good in Hydrogen...
image

Comment (kylebarron) [reply stata_kernel/docs/css/pandoc.css]: I don't see a difference in the Notebook

2em:
image

0em:
image

Comment (kylebarron) [reply stata_kernel/docs/css/pandoc.css]: Oh nevermind, I have to recompile each time...

Comment (kylebarron) [reply stata_kernel/docs/css/pandoc.css]: Even after recompiling it still looks fine with `0em` in the Notebook and better in Hydrogen:
image
image

**Comment by kylebarron** [comment]
This changed the color of the repo bar to HTML :cry: 

image

**Comment by mcaceresb** [comment]
Seems like a github bug...clearly not 98% HTML.

**Pull request merged**